cohort-engine/src/main/java/com/ibm/cohort/engine/translation/InJVMCqlTranslationProvider.java

```diff
@@ -120,4 +100,9 @@ public Library translate(InputStream cql, List<Options> options, LibraryFormat t
 		return result;
 	}
 
+	@Override
+	public ModelInfo convertToModelInfo(File modelInfoObject) {
+		return JAXB.unmarshal(modelInfoObject, ModelInfo.class);
+	}
+
```

Comment:
I'm not opposed to generics, but I'm not sure this is the best use. I'd like to see several overloads that take File, InputStream, and Reader vs. using the generics for this specific.